src/services/oss/backend.rs

```diff
@@ -0,0 +1,977 @@
+    }
+
+    async fn oss_complete_multipart_upload(
+        &self,
+        path: &str,
+        upload_id: &str,
+        parts: &[ObjectPart],
+    ) -> Result<Response<IncomingAsyncBody>> {
+        let p = build_abs_path(&self.root, path);
+        let url = format!(
+            "{}/{}?uploadId={}",
+            self.endpoint,
+            percent_encode_path(&p),
+            upload_id
+        );
+        let content = se::to_string(&CompleteMultipartUploadRequest {
+            part: parts
+                .iter()
+                .map(|v| CompleteMultipartUploadRequestPart {
+                    part_number: v.part_number(),
+                    etag: v.etag().to_string(),
+                })
+                .collect(),
+        })
+        .map_err(|e| {
+            new_other_object_error(
+                Operation::CompleteMultipart,
+                path,
+                anyhow::anyhow!("build xml: {:?}", e),
+            )
+        })?;
+
+        let length = content.len();
+        let body = AsyncBody::Bytes(Bytes::from(content));
+        let mut req = Request::post(url)
+            .header(CONTENT_LENGTH, length)
+            .header(CONTENT_TYPE, "application/xml")
+            .body(body)
+            .map_err(|e| new_request_build_error(Operation::CompleteMultipart, path, e))?;
+
+        self.signer
+            .sign(&mut req)
+            .map_err(|e| new_request_sign_error(Operation::CompleteMultipart, path, e))?;
+        self.client
+            .send_async(req)
+            .await
+            .map_err(|e| new_request_send_error(Operation::CompleteMultipart, path, e))
+    }
+
+    async fn oss_abort_multipart_upload(
+        &self,
+        path: &str,
+        upload_id: &str,
+    ) -> Result<Response<IncomingAsyncBody>> {
+        let p = build_abs_path(&self.root, path);
+        let url = format!(
+            "{}/{}?uploadId={}",
+            self.endpoint,
+            percent_encode_path(&p),
+            upload_id
+        );
+        let mut req = Request::delete(url)
+            .body(AsyncBody::Empty)
+            .map_err(|e| new_request_build_error(Operation::AbortMultipart, path, e))?;
+        self.signer
+            .sign(&mut req)
+            .map_err(|e| new_request_sign_error(Operation::AbortMultipart, path, e))?;
+
+        self.client
+            .send_async(req)
+            .await
+            .map_err(|e| new_request_send_error(Operation::AbortMultipart, path, e))
+    }
+}
+
+#[async_trait]
+impl Accessor for Backend {
+    fn metadata(&self) -> AccessorMetadata {
+        let mut am = AccessorMetadata::default();
+        am.set_scheme(Scheme::Oss)
+            .set_root(&self.root)
+            .set_name(&self.bucket)
+            .set_capabilities(
+                AccessorCapability::Read
+                    | AccessorCapability::Write
+                    | AccessorCapability::List
+                    | AccessorCapability::Multipart,
```

Comment:
No. Not in this PR.

I will reject all PRs trying to add more features before OSS passes our read/write integration tests.